oak_functions/client/android/com/google/oak/functions/android/client/MainActivity.java

```diff
@@ -30,24 +29,12 @@
 import com.google.oak.functions.android.client.R;
 import com.google.oak.functions.client.AttestationClient;
 import com.google.protobuf.ByteString;
-import java.io.BufferedReader;
-import java.io.InputStreamReader;
-import java.lang.Runnable;
-import java.lang.ref.WeakReference;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.Optional;
-import java.util.stream.Collectors;
 import oak.functions.invocation.Request;
 import oak.functions.invocation.Response;
 import oak.functions.invocation.StatusCode;
 
 /** Main class for the Oak Functions Client application. */
 public class MainActivity extends Activity {
-  private HandlerThread backgroundHandlerThread;
```

Comment:
So these fields were here but never used?!

Reply:
Yes, I think it was a copy-paste from the Hello World Android client, which does the service call in the background. This one does not yet do it. I have created #2204 to track moving this logic to a background thread.